src/app/pages/scans/scans-management/scan-observable/scan-observable.component.ts

```diff
@@ -25,6 +26,7 @@ export class ScanObservableComponent {
   onObsClassificationChange(): void {
     // Clear selected analyzers on classification change
     this.formData.analyzers_requested = [];
+    this.formData.connectors_requested = [];
```

Comment:
not required. We reset the analyzers_requested because the options list changes on classification change. The same does not apply to connectors.